msctl

```diff
@@ -1375,11 +1375,7 @@ overviewer() {
           },
           {
             "text" : "Overviewer",
-            "color" : "aqua",
-            "clickEvent" : {
-              "action" : "open_url",
-              "value" : "'$OVERVIEWER_URL'"
-            }
+            "color" : "aqua"
           },
```

Comment:
My bad, I had dropped it on my own local server because I didn't really think it was necessary, but we should leave it